modules/ROOT/pages/splitter-flow-control-reference.adoc

```diff
@@ -444,6 +444,14 @@ INFO  YYYY-DD-MM HH:MM:SS,SSS [] org.mule.api.processor.LoggerMessageProcessor:
 INFO  YYYY-DD-MM HH:MM:SS,SSS [] org.mule.api.processor.LoggerMessageProcessor: [[#text: Liam Neeson], [#text: Will Ferrell], [#text: Christian Bale]]
 ----
 
+== Nesting Splitters
```

Comment:
Should this concept topic precede the example?

Reply:
Also, should we say not only what these are and what they do, but WHY the audience wants or needs to use them?